docker-hub/image-management.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Advanced Image Management dashboard
+keywords: dashboard, images, image management, inactive
+title: Advanced Image Management dashboard
+---
+
+> The **Advanced Image Management** dashboard allows you to manage Docker images across all repositories. [Upgrade](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"} your existing account to a Pro or a Team account to start using the Advanced Image Management dashboard.
+
+The Advanced Image Management dashboard provides a snapshot of your existing images and allows you to view, sort and filter images by tags, activity status and date, and contains options to clean up your workspace by deleting images that are no longer required.
+
+The dashboard provides visibility into the status of the images in your repository and allows you to reduce the number of images stored in Docker Hub by reviewing their active status. An image retains its 'active' status if it is pulled or pushed in the last month. If there isn’t any activity on the image in the last month, it is considered 'inactive'.
+
+By displaying the activity status and tags, the dashboard makes it easier for you to review the existing images and take actions to delete images that are not needed anymore. This is particularly helpful when your repository contains a lot of images that you have pushed over time, and you are now trying to streamline storage in Docker Hub.
```

Comment:
add in:

You can also see the old versions of images you have pushed. When you push an image to Docker Hub you are pushing a manifest, a list of all of the layers of your image, and the layers themselves. When you are updating an existing tag, only the new layers will be pushed along with the new manifest which references these layers. This new manifest will be given the tag you specify when you push, such as myNamespace/mytag:latest. This does not remove the old manifests or the unique layers referenced by them from Hub.
You can in still use and reference these using the digest of the manifest if you know the SHA.

With the new Advanced Image management Dashboard all of this is now accessible for you to browse, use and clean up.